Utils/azuretools-core/src/com/microsoft/azuretools/telemetry/AppInsightsClient.java

```diff
@@ -186,7 +186,12 @@ private static void initTelemetryManager() {
         try {
             TelemetryManager.getInstance().setCommonProperties(buildProperties("", new HashMap<>()));
             TelemetryManager.getInstance().setTelemetryClient(TelemetryClientSingleton.getTelemetry());
-            TelemetryManager.getInstance().setEventNamePrefix(configuration.eventNamePrefix());
+            // The eclipse eventName prefix is AzurePlugin.Eclipse. We should remove the last "."
```

Comment:
What don't we change the event name to `AzurePlugin.Eclipse`?

Reply:
Add a new method eventName()